cas-server-webapp/src/main/webapp/WEB-INF/cas.properties

```diff
@@ -781,3 +781,19 @@ accept.authn.users=casuser::Mellon
 # cas.mfa.request.parameter=authn_method
 # cas.mfa.failure.mode=PHANTOM
 # cas.mfa.authn.ctx.attribute=authnContextClass
+
+#CAS components mappings
+primaryAuthenticationHandler=acceptUsersAuthenticationHandler
+primaryPrincipalResolver=personDirectoryPrincipalResolver
```

Comment:
It's done as usual by mapping #{null} in deployerConfigContext. Has nothing to do with aliases. This is just a convenience of externalizing the current aliases mapping so folks don't have to deal with yet more XML dancing when enabling different modules relying on these aliases.